sdks/go/test/integration/transforms/xlang/dataframe/dataframe_test.go

```diff
@@ -0,0 +1,60 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dataframe
+
+import (
+	"flag"
+	"log"
+	"testing"
+
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/dataflow"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/flink"
+	_ "github.com/apache/beam/sdks/v2/go/pkg/beam/runners/universal"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/testing/ptest"
+	"github.com/apache/beam/sdks/v2/go/test/integration"
+)
+
+var expansionAddr string // Populate with expansion address labelled "python_transform".
+
+func checkFlags(t *testing.T) {
+	if expansionAddr == "" {
+		t.Skip("No python transform expansion address provided.")
+	}
+}
+
+func TestDataframe(t *testing.T) {
+	integration.CheckFilters(t)
+	checkFlags(t)
+	p := DataframeTransform(expansionAddr)
+	ptest.RunAndValidate(t, p)
+}
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	beam.Init()
+
+	services := integration.NewExpansionServices()
+	defer func() { services.Shutdown() }()
+	addr, err := services.GetAddr("python_transform")
```

Comment:
I'm assuming the initialization of `python_transform` is in another PR?

The portable tests skip this:

```
2022/10/05 16:08:29 skipping missing expansion service: expansion service labeled "python_transform" not found: no --expansion_jar or --expansion_addr flag provided with label "python_transform"
=== RUN   TestDataframe
    dataframe_test.go:35: No python transform expansion address provided.
```

Reply:
It'll be added in a separate PR